### PR TITLE
L10n

### PR DESCRIPTION
**Pull request by snaggen**
Here is a pull request for supporting multiple languages, I have added the most common languages based on strftime abbrs with different locales. As usual I haven't hooked up the preferences for this, so that still needs to be done. 
I have also added some fixes to the weather resource switch statement, some of the conditions were never used.

### CODE REVIEW

File: futura_weather_redux/src/main.c [hunk 1]
```diff
@@ -356,8 +358,12 @@ void handle_tick(struct tm *now, TimeUnits units_changed) {
     }
     
     if(units_changed & DAY_UNIT) {
-        static char date_text[11];
-        strftime(date_text, 11, "%a %b %d",  now);
+        /* The length 18 should be enougth to fit the most common language abbr */
+        static char date_text[18];
+        /*TODO: Read lang from preferences, for lang numbers se futura_lang.h */
+        int lang = 0; //Hardcoded to en_US until pref is in place
```
Comment (snaggen) [inline futura_weather_redux/src/main.c:364]: This lang value should be read from some preferences

File: futura_weather_redux/src/js/pebble-js-app.js [hunk 2]
```diff
@@ -93,7 +210,15 @@ Pebble.addEventListener("appmessage", function(e) {
 			if(e.payload[key] !== "undefined") { prefs[key] = e.payload[key]; }
 	}
 	else if(e.payload["requestWeather"] == 1) {
-		fetchWeather();
+        if(Math.round(Date.now()/1000) - weather.lastUpdate >= maxWeatherUpdateFreq) {
+            //TODO: Check pref and call the correct weather service
```
Comment (snaggen) [inline futura_weather_redux/src/js/pebble-js-app.js:214]: This is were the preferences should be used to determine which provider to use.

**Comment by kylewlacy** [comment]
I'm not going to merge this branch; the plan is to add the actual translations to the `preferences.htm` file under the `gh_pages` branch. That page will then pass each day/month name to the `pebble-js-app.js`, which will then transmit each to the watch. This has 2 distinct advantages over this solution:
1. New translations and languages can be added without recompiling the binary
2. Translations can be added and fixed 'remotely'; since the preference page is accessed live from the `gh_pages` branch, any commits will appear 'live' to any users

I think I will use these translations in that version. You don't mind if I copy/paste that hash in your `futura_lang.h`? It won't technically be a 'merge' by GitHub's metric.

**Comment by snaggen** [comment]
Yes, please use my translations. And when you redo this you may want to
sort the languages in the drop down menu.
And instead of cherry picking, it is probably easier to just merge every
thing and then  change the two lang related lines in main.c and remove the
futura_lang.h file.

/Snaggen
Den 25 jan 2014 02:57 skrev "Kyle Lacy" notifications@github.com:

**Pull request closed without merging**